packages/react-cookie/src/server.ts

```diff
@@ -0,0 +1,12 @@
+import {Context} from 'koa';
+import {Cookie} from './types';
+
+export function applyCookieToContext(cookies: Cookie) {
+  return function(ctx: Context) {
+    Object.entries(cookies).forEach(([cookie, options]) => {
+      const {value, ...cookieOptions} = options;
+
+      ctx.cookies.set(cookie, value, cookieOptions as any);
```

Comment:
Could you add `none` in a PR to https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cookies/index.d.ts#L91, please?